_posts/2019-02-22-indico-2-2-news.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title:  "Introducing Indico 2.2 - featuring completely new room booking system!"
+date:   2019-02-21 10:00
+author: "Indico Team"
+categories: indico update release milestone
+---
+
+Indico 2.2 is coming soon and we would like to introduce the new features to you. If you're using Indico to manage room bookings, we are sure that you will particularly appreciate this release.
+Organizers of events and conferences always need to take care of booking spaces with specific requirements. We wanted to make this task much easier and better connected with Indico events.
+
+Instead of a regular description of what changed between versions of the system, we thought we would walk you through the new system.
```

Comment:
that sounds a bit strange IMO - isn't it clear that this is what such a blog post does?

Reply:
I think the point is to say that instead of diff of features, this is more like an introduction. But it's a bit hard to explain. Neither the original version nor my changes seem to do a good job at it 😢 
Maybe we should just remove that bit?

Reply:
/cc @meluru

Reply:
yeah, I agree